### PR TITLE
UI of view Projects Enhanced

### LINKED ISSUE #180: UI of view Projects
I would like to give it a try to make ui of view projects  better.

**Note : The image is not saving in db so i can't render it on view project page . it will show image not found or i can hard code the image part** 

I can make a better ui for The data currently on that page using bootstrap cards 
It will look something like this.
ui of cards

### PR DESCRIPTION
**Pull request by arpit456jain**
# Description:

 UI of view Projects Enhanced

Fixes #180 

https://user-images.githubusercontent.com/55352601/118357159-4f97c380-b596-11eb-95cb-8e92ae3f5389.mp4

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

 
# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

# Screenshots / Video:

### CODE REVIEW

File: templates/modules.html
Comment (Aryamanz29) [inline templates/modules.html:14]: Hii 👋 @arpit456jain 
Could you please move your page's internal CSS  to `css/moduleStyle.css`?

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

File: templates/modules.html
Comment (Aryamanz29) [inline templates/modules.html:14]: Hii 👋 @arpit456jain 
Could you please move your page's internal CSS  to `css/moduleStyle.css`?

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

File: templates/modules.html
Comment (arpit456jain) [reply templates/modules.html:14]: Yes i can but let it ui be approved first other wise once again merge conflicts occurs 
I had made theses changes 3rd time

**Comment by arpit456jain** [comment]
@Aryamanz29 I had moved all internal css to modeule.css and done some edits now its like this

https://user-images.githubusercontent.com/55352601/118369069-9e5a5300-b5c0-11eb-801e-3c7fc5d83c99.mp4

**Comment by arpit456jain** [comment]
@SaiSudhaV @sruti2024 please review this PR

**Review by Aryamanz29 — APPROVED** [review]

**Comment by Aryamanz29** [comment]
@arpit456jain  LGTM. Thanks for contribution 🎉

**Comment by arpit456jain** [comment]
Lets see which label i will got 😅

**Comment by arpit456jain** [comment]
Thanks @Aryamanz29

**Comment by SaiSudhaV** [comment]
Hi @arpit456jain
Can you design the page more attractive? You can have the project details card aligned to the center and below that display the table containing sub-projects and tasks to be done

**Comment by arpit456jain** [comment]
> Hi @arpit456jain
> Can you design the page more attractive? You can have the project details card aligned to the center and below that display the table containing sub-projects and tasks to be done

@SaiSudhaV  i had  done that before you can check my first my PR but @sruti2024 asked me to display like this 😅🤔

**Comment by arpit456jain** [comment]
@SaiSudhaV  have a look on #187

**Comment by arpit456jain** [comment]
I can't make it more attractive than this currently .

**Comment by SaiSudhaV** [comment]
Okay that's fine

**Comment by arpit456jain** [comment]
@SaiSudhaV @sruti2024 kindly review my PR

**Pull request merged**
